mobi OSD

Developer-felix opened the pull request:
The OSD should be displayed in two lines on mible

michellemerrill committed:
Create README.md (#8554)

* Create README.md

Adding in `Code Search` category

* Update README.md

Changing code search link

* Update README.md

Co-authored-by: David Staheli <1767415+davidstaheli@users.noreply.github.com>

Co-authored-by: David Staheli <1767415+davidstaheli@users.noreply.github.com>

michellemerrill committed:
Update README.md (#8619)

Updating Lists linkage

michellemerrill committed:
Update README.md (#10931)

Adding `dependabot` category to the readme

michellemerrill committed:
Update README.md (#12296)

Deleting `Releases` category, will absorb into the `General` category

michellemerrill committed:
Update README.md (#12684)

Adding category finder in table view

natalierjackson committed:
update Readme.md to include feedback category

natalierjackson committed:
text change

natalierjackson committed:
Merge pull request #13365 from github/natalierjackson-feed-feedback-readme-update

update Readme.md to include feedback category

natalierjackson committed:
update readme 

Code Navigation team requested a new category name

natalierjackson committed:
Update README.md

natalierjackson committed:
Update README.md

natalierjackson committed:
Merge pull request #14513 from github/natalierjackson-feedback-update-1

update readme

natalierjackson committed:
Update README.md

natalierjackson committed:
Merge pull request #16020 from github/natalierjackson-patch-1

Update README.md for rename of dependabot category

michellemerrill committed:
Update README.md

Adding `accessibility` category to the readme

natalierjackson committed:
Merge pull request #16937 from github/michellemerrill-patch-3

Update README.md

natalierjackson committed:
adding new url

natalierjackson committed:
update url

natalierjackson committed:
Update README.md

natalierjackson committed:
Update README.md

natalierjackson committed:
update readme

natalierjackson committed:
Merge pull request #17838 from github-community/natalie/update-readme-orbit

Updating Readme URL's

ettaboyle committed:
Update readme icon for education

Updating the icon for education from a book to the backpack (satchel)

ettaboyle committed:
Merge pull request #18044 from github-community/education-icon-update

Update readme icon for education

tuves committed:
fixed feedback link

natalierjackson committed:
Update README.md

tuves committed:
Merge pull request #18747 from github-community/tuves-patch

updated feedback link

Developer-Felix commented:
Can You please review my PR

346512529 reviewed (commented):
Akawnt facebook on Meta Tar Ani

Mystic421980 reviewed (commented):
SigTerm

tuves committed:
update readme

tuves committed:
Merge pull request #23631 from community/CoC-patch

update readme

ettaboyle committed:
Update code_of_conduct.md link

Updated from the Community Covenant draft to instead link to the GitHub Community Forum Code of Conduct in the GitHub docs
* No preview available as this only updates the link on the homepage

michellemerrill committed:
Update CODE_OF_CONDUCT.md (#25245)

Live review with @ettaboyle

michellemerrill committed:
Update CODE_OF_CONDUCT.md

Adding full CoC language
